### PR TITLE
fixed feature feedback screen

### PR DESCRIPTION
**Pull request by lordpacificsmartly**
Fix Feature Feedback screen

Implement Epoxy RecyclerView for the feedback fragment

#### Checklist
- [x] Unit tests
- [x] UI Test
- [x] Comment
- [x] Documentation

#### Screenshots
Original                        |   Updated
:------------------------------:|:------------------------------:
    **  original screenshot **  |   
Feedback Screen

### CODE REVIEW

**Comment by lordpacificsmartly** [comment]
@darothub  PTAL

**Comment by darothub** [comment]
> @darothub PTAL

Nice job you have done.  Could you make use one row item have elevation as it is in the UI.

File: app/src/main/java/ng/com/thewhitecellfoundation/haemcam/ui/home/FeedbackFragment.kt [hunk 1]
```diff
@@ -15,7 +19,38 @@ import ng.com.thewhitecellfoundation.haemcam.databinding.FragmentFeedbackBinding
...
+    @SuppressLint("ResourceType")
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+
+        val categories = resources.getStringArray(R.array.categories)
+        val arrayAdapter = ArrayAdapter(requireContext(), R.layout.fragment_feedback_dropdown_item, categories)
+        binding.fragmentFeedbackAutocompleteTextview.setAdapter(arrayAdapter)
+
+        val name = "Dr Anthonia Peterson"
+        val message = "Here are some medical feedback.... " + "Here are some medical feedback....Here are some medical feedback....  " + " Here are some medical feedback...."
```
Comment (darothub) [inline app/src/main/java/ng/com/thewhitecellfoundation/haemcam/ui/home/FeedbackFragment.kt:36]: Do not hard code the ellipsis(...) at the end of the text.

**Review by darothub — CHANGES_REQUESTED** [review]

File: app/src/main/java/ng/com/thewhitecellfoundation/haemcam/ui/home/FeedbackFragment.kt (hunk 1 above)
Comment (lordpacificsmartly) [reply app/src/main/java/ng/com/thewhitecellfoundation/haemcam/ui/home/FeedbackFragment.kt:36]: Okay sir. Making corrections now

**Pull request merged**